justice/xform.py

```diff
@@ -39,6 +40,21 @@ def make_lc2d(self, lc: lightcurve._LC) -> lightcurve.LC2D:
         detected = np.concatenate([band.detected for name, band in lc.bands.items()])
         return lightcurve.LC2D(pwav, time, flux, flux_err, detected)
 
+    def transform_band(self, bd, ty, dy):
```

Comment:
make sure you actually use this data? i thought we had similar code for this functionality.

maybe use longer names or type annotations, e.g. `bd: BandData`.

Reply:
I think this got somehow readded when it should have been deleted